test/src/rabbit_tests.erl

```diff
@@ -3463,3 +3463,34 @@ disconnect_detected_during_alarm([ACfg]) ->
     [] = ListConnections(),
 
     passed.
+
+
+rabbitmqctl_list_consumers(Config) ->
+    StdOut = rabbit_test_configs:rabbitmqctl(Config, "list_consumers"),
+    [<<"Listing consumers", _/binary>> | ConsumerRows] = re:split(StdOut, <<"\n">>, [trim]),
+    CTags = [ lists:nth(3, re:split(Row, <<"\t">>)) || Row <- ConsumerRows ],
+    CTags.
+
+list_consumers_sanity_check_with() ->
+    start_and_connect_a.
+
+list_consumers_sanity_check([ACfg]) ->
+    Chan = pget(channel, ACfg),
+    #'queue.declare_ok'{} = amqp_channel:call(Chan, #'queue.declare'{queue = <<"abc">>}),
```

Comment:
This test heavily relies on `_with` testing framework, which starts separate node(s) for each such test. So cleanup will happen automatically just by destroying that instance of the broker.

It was easier to write that test that way, because e.g. with `tests-with-broker` some additional code is needed to be sure that stray queue from some other test doesn't interfere and result in false positive.

So should I amend this patch with autogenerated queue name or it is good the way it is now?